### PR TITLE
PAB-3836: Renaming of Use button

### PR DESCRIPTION
**Pull request by kailol-sag**
Need to ask Gyan via iTrac to review this.

### CODE REVIEW

File: content/streaming-analytics/analytics-builder-bundle/using-the-instance-editor.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ The instance editor table provides different types of input controls, depending
...
 -   Drop-down list boxes are provided when you can select a different value \(for example, to select a different rule for rounding\).
--   When you edit a device or group of devices, an additional dialog box appears. The dialog box is the same as when selecting a different device or group of devices in the block parameter editor \(see [Editing the parameters of a block](/streaming-analytics/analytics-builder/#editing-the-parameters-of-a-block) for more information on this dialog box\). Click the **Use** button \(this is shown when you move the mouse over an entry\) to select the device or group of devices that you want to use.
+-   When you edit a device or group of devices, an additional dialog box appears. The dialog box is the same as when selecting a different device or group of devices in the block parameter editor \(see [Editing the parameters of a block](/streaming-analytics/analytics-builder/#editing-the-parameters-of-a-block) for more information on this dialog box\). Click the button which is shown when you move the mouse over an entry to select the device or group of devices that you want to use.
```
Comment (kailol-sag) [inline content/streaming-analytics/analytics-builder-bundle/using-the-instance-editor.md:57]: Gyan, why do we not mention the assets in the doc for the instance editor? Has this been forgotten to add?

Comment (gyan-sag) [reply content/streaming-analytics/analytics-builder-bundle/using-the-instance-editor.md:57]: Do you mean in this paragraph itself or somewhere else? Yes, I think we should mention asset in this paragraph wherever we are talking about "device or group of devices".

Comment (kailol-sag) [reply content/streaming-analytics/analytics-builder-bundle/using-the-instance-editor.md]: Fixed the wording in the instance editor topic to also mention assets.

File: content/streaming-analytics/analytics-builder-bundle/using-the-model-editor.md [hunk 2]
```diff
@@ -139,16 +139,28 @@ For the input and output blocks, you can globally replace the devices, groups or
...
 
-    Click the **Use** button \(this is shown when you move the mouse over an entry\) to select the device, group or asset that you want to use.
+    Click the button which is shown when you move the mouse over an entry to select the device, group or asset that you want to use.
+    The name of that button depends on the type of source that is currently selected:
+
+    | Button name | Shown for | Description |
+    | ----------- | --------- | ----------- |
+    | **Choose device** | Devices | Data is received from the device or sent to the device. |
+    | **Choose group's devices** | Groups in input blocks | Configures the block to run for all devices within the group hierarchy. You cannot directly receive from a group. |
+    | **Choose asset** | Assets in input blocks | Receive the asset itself. The devices of the assets are ignored. |
```
Comment (gyan-sag) [inline content/streaming-analytics/analytics-builder-bundle/using-the-model-editor.md:155]: I am not sure "Receive the asset itself." makes sense. May be "Data is received from the asset itself."

File: content/streaming-analytics/analytics-builder-bundle/getting-started.md [hunk 3]
```diff
@@ -66,7 +66,7 @@ You design your model in the model editor. The model editor is shown after you h
 If the block parameter editor is not shown (for example, because you clicked an empty space on the canvas after dragging the input block onto the canvas), click the block using the left mouse button to show the block parameter editor.
     {{< /c8y-admon-info>}}
 
-3.  Click the three dots that are shown for **Input Source**. In the resulting dialog box, click the **Use** button to select your device. This button is shown when you move the mouse over a device.
+3.  Click the three dots that are shown for **Input Source**. In the resulting dialog box, click the **Choose device** button to select your device. This button is shown when you move the mouse over a device.
```
Comment (BeateRixen) [inline content/streaming-analytics/analytics-builder-bundle/getting-started.md:69]: It is "Trigger Device" below (with a capital "D"), not sure what is correct

File: content/streaming-analytics/analytics-builder-bundle/using-the-model-editor.md [hunk 4]
```diff
@@ -139,16 +139,28 @@ For the input and output blocks, you can globally replace the devices, groups or
...
+    | **Choose asset's devices** | Assets in input blocks | Run for all devices within the asset hierarchy. The block does not receive from the asset. |
+    | **Choose asset** | Assets in output blocks | Send output to the asset itself. |
+
+    For the output blocks, you can also select a **Trigger Device**. This sends the output to the device which triggered the output.
```
Comment (BeateRixen) [inline content/streaming-analytics/analytics-builder-bundle/using-the-model-editor.md:159]: **Trigger Device** > trigger device

File: content/streaming-analytics/analytics-builder-bundle/using-the-model-editor.md [hunk 5]
```diff
@@ -356,7 +368,7 @@ After you have replaced the devices, you need to verify that the measurements th
 
 2.  In the **Current device, group or asset** drop-down list box of the resulting dialog box, select the device, group or asset that you want to replace. All devices, groups and assets that are used in the model are available for selection.
 
-3.  Click the **Replace with** box to display a dialog box. The dialog box is the same as when selecting a different device, group or asset in the block parameter editor. See [Editing the parameters of a block](/streaming-analytics/analytics-builder/#editing-the-parameters-of-a-block) for more information on this dialog box. Click the **Use** button \(this is shown when you move the mouse over an entry\) to select the device, group or asset that you want to use instead.
+3.  Click the **Replace with** box to display a dialog box. The dialog box is the same as when selecting a different device, group or asset in the block parameter editor. See [Editing the parameters of a block](/streaming-analytics/analytics-builder/#editing-the-parameters-of-a-block) for more information on this dialog box. Click the button which is shown when you move the mouse over an entry to select the device, group or asset that you want to use instead.
```
Comment (BeateRixen) [inline content/streaming-analytics/analytics-builder-bundle/using-the-model-editor.md:371]: when you move the mouse over > perhaps "hover over"?

File: content/streaming-analytics/analytics-builder-bundle/getting-started.md (hunk 3 above)
Comment (kailol-sag) [reply content/streaming-analytics/analytics-builder-bundle/getting-started.md:69]: This is how it's shown in the UI.
image

File: content/streaming-analytics/analytics-builder-bundle/using-the-model-editor.md (hunk 2 above)
Comment (kailol-sag) [reply content/streaming-analytics/analytics-builder-bundle/using-the-model-editor.md]: I've update this, and also made some more changes to make the wording in the table a bi tmore consistent.

File: content/streaming-analytics/analytics-builder-bundle/using-the-model-editor.md (hunk 5 above)
Comment (kailol-sag) [reply content/streaming-analytics/analytics-builder-bundle/using-the-model-editor.md]: Changed this here and also in several similar places.

**Review by BeateRixen — APPROVED** [review]

**Review by gyan-sag — APPROVED** [review]
everything seems fine to me

**Pull request merged**